.github/workflows/build.yml

```diff
@@ -0,0 +1,53 @@
+name: Build React App
```

Comment:
Give a proper name for workflow like World Clock and also give a different name for the file as it will gives conflict with another one